### PR TITLE
Core v0.2

### LINKED ISSUE #5: Update core to v0.2

### PR DESCRIPTION
**Pull request by mat-hek**
closes #5

### CODE REVIEW

File: c_src/membrane_element_portaudio/sink.spec.exs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+       pa_buffer_size :: int,
+       latency :: atom
+     ) :: {:ok :: label, state} | {:error :: label, reason :: atom}
+
+spec write_data(payload, state) :: (:ok :: label) | {:error :: label, :overrun :: label}
+
+spec destroy(state) :: :ok
+
+sends {Native, :destroy :: label, state}
+sends {:demand :: label, size :: int}
```
Comment (bblaszkow06) [inline c_src/membrane_element_portaudio/sink.spec.exs]: Isn't that atom a bit too generic?

File: lib/membrane_element_portaudio/sink.ex [hunk 2]
```diff
@@ -45,62 +46,55 @@ defmodule Membrane.Element.PortAudio.Sink do
...
   end
 
   @impl true
-  def handle_other(
-        {:membrane_element_portaudio_ringbuffer_demand, size},
-        %{playing: true} = state
-      ) do
-    {{:ok, demand: {:sink, size}}, state}
+  def handle_other({:demand, size}, %{playback_state: :playing}, state) do
+    {{:ok, demand: {:input, &(&1 + size)}}, state}
```
Comment (bblaszkow06) [inline lib/membrane_element_portaudio/sink.ex]: In `sink.c`(L31) you have already changed demand calculation to include the demand. Here you increase the demand again. Is that an error or am I missing something?

File: mix.exs [hunk 3]
```diff
@@ -1,15 +1,15 @@
 defmodule Membrane.Element.PortAudio.Mixfile do
   use Mix.Project
-  Application.put_env(:bundlex, :membrane_element_portaudio, __ENV__)
 
   @github_url "https://github.com/membraneframework/membrane-element-portaudio"
+  @version "0.1.0"
```
Comment (bblaszkow06) [inline mix.exs]: Don't forget to bump the version

File: test/membrane_element_portaudio/sink_test.exs [hunk 4]
```diff
@@ -12,52 +13,52 @@ defmodule Membrane.Element.Portaudio.SinkTest do
...
   def playing(%{state: state}) do
-    %{state: %{state | native: make_ref(), playing: true}}
+    %{state: %{state | native: make_ref()}}
   end
 
   setup_all :state
 
-  describe "handle_play" do
+  describe "handle_prepared_to_playing" do
+    @tag skip: "Temporairly disabled due to mocking issues"
```
Comment (bblaszkow06) [inline test/membrane_element_portaudio/sink_test.exs]: Please check if we can do something about it by placing mockable macro inside the SyncExexutor module

Comment (mat-hek) [reply test/membrane_element_portaudio/sink_test.exs]: Unfortunately, it doesn't work. It seems that mockery is based on the per-process dictionary.

**Pull request merged**